dev/io.openliberty.jaxb.3.0.internal.tools/src/io/openliberty/jaxb/tools/internal/JaxbToolsConstants.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jaxb.tools.internal;
+
+import java.util.ResourceBundle;
+
+public class JaxbToolsConstants {
+    public static final String PARAM_HELP = "-help";
+
+    public static final String PARAM_VERSION = "-version";
+
+    public static final String PARAM_TARGET = "-target";
+
+    public static final String TR_GROUP = "JaxbTools";
+
+    public static final String TR_RESOURCE_BUNDLE = "io.openliberty.jaxb.tools.internal.resources.JaxbToolsMessages";
```

Comment:
You changed the name here, but you did not change the file directory structure in the resources directory.